CHANGELOG.md

```diff
@@ -1,10 +1,48 @@
 # Changelog
 
-## main / unreleased
-
-* [BUGFIX] OTLP receiver: Generate `target_info` samples between the earliest and latest samples per resource. #16737
-* [BUGFIX] Config: Infer escaping scheme when scrape config validation scheme is set.
+## 3.7.0-rc.0 / 2025-10-02
+
+* [FEATURE] PromQL: Add support for experimental anchored and smoothed rate. #16457
+* [FEATURE] Federation: Add support for native histograms with custom buckets (NHCB). #17215
+* [FEATURE] PromQL: Add `first_over_time(...)` and `ts_of_first_over_time(...)` behind feature flag `experimental-promql-functions`. #16963 #17021
+* [FEATURE] Remote-write: Add support for Azure Workload Identity as an authentication method for the receiver. #16788
+* [FEATURE] Remote-write: Add type and unit labels to outgoing time series in remote-write 2.0 when the `type-and-unit-labels` feature flag is enabled. #17033
+* [FEATURE] OTLP: Write start time of metrics as created time zero samples into TSDB when `created-timestamp-zero-ingestion` feature flag is enabled. #16951
+* [ENHANCEMENT] PromQL: Add warn-level annotations for counter reset conflicts in certain histogram operations. #17051 #17094
+* [ENHANCEMENT] PromQL: Test scripting language supports instant queries which return a string literal or a range vector result. #17055
+* [ENHANCEMENT] UI: Add scrape interval and scrape timeout to targets page. #17158
+* [ENHANCEMENT] TSDB: Reduce the resolution of native histograms read from chunks or remote read if the schema is exponential. #17213
+* [ENHANCEMENT] Remote-write: Add logging and metrics for unexpected metadata in samples batches. When metadata entries are found in samples-only remote-write batches. #17034
+* [ENHANCEMENT] Remote write: Add logging for unexpected metadata in sample batches, when metadata entries are found in samples-only remote write batches. #17082
```

Comment:
```suggestion
* [ENHANCEMENT] Remote write: Add logging for unexpected metadata in sample batches, when metadata entries are found in samples-only batches. #17082
```

Reply:
I've added #17034 to this line and removed the line for #17034, apparently the metric was not added in the end.